site/tests/universal/homepage.test.ts

```diff
@@ -1,5 +1,12 @@
 import { expect, test } from "@playwright/test";
 
+const links = {
+  exploreAllBlocks: "/hub",
+  developingBlocks: "/docs/developing-blocks",
+  embeddingBlocks: "/docs/embedding-blocks",
+  login: "/login",
+};
```

Comment:
I'd just hardcode the values in assertion statements. The fewer levels of abstraction we have in tests, the more robust they are. Repetition is fine – we can always use search & replace if we need it.

Reply:
Sounds good. Updated here => https://github.com/blockprotocol/blockprotocol/pull/443/commits/54e97e8463a588161ddf075be0d38afd4331ced6